CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to Archive of Our Own
```

Comment:
Sounds good to me! I just wanted to check given we're still talking about OTW-Archive software and the Archive separately in the readme.